client/blocks/login/login-form.jsx

```diff
@@ -801,24 +815,6 @@ export class LoginForm extends Component {
 								/>
 							) }
 
-							{ isWoo && linkingSocialUser && (
-								<Notice
-									className="login__form-user-exists-notice"
-									status="is-warning"
-									icon={ <Icon icon={ alert } size={ 20 } fill="#d67709" /> }
-									showDismiss
-									onDismissClick={ this.props.cancelSocialAccountConnectLinking }
-									text={ this.props.translate(
-										'You already have a WordPress.com account with this email address. Add your password to log in or {{signupLink}}create a new account{{/signupLink}}.',
-										{
-											components: {
-												signupLink: <a href={ signupUrl } />,
-											},
-										}
-									) }
-								/>
-							) }
-
```

Comment:
This was added by https://github.com/Automattic/wp-calypso/pull/87042, but it looks like that flow is not used anymore. I've decided to clean this up and show the wpcom notice (`renderSocialLinkingNotice`) instead.

Reply:
Got it. Thanks for confirming!